src/OMSimulatorLua/OMSimulatorLua.c

```diff
@@ -1309,5 +1421,13 @@ DLLEXPORT int luaopen_OMSimulatorLua(lua_State *L)
   REGISTER_LUA_CALL(omsi_solve);
 #endif /* WITH_OMSYSIDENT */
 
+  // Add enumerable constants
+  lua_pushnumber(L, 0);
```

Comment:
I guess it is better to use the enum directly, e.g. `oms_causality_input`, just in case that the order changes in the future for some reason.